bundles/org.openhab.binding.mikrotik/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.mikrotik</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Mikrotik Binding</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>me.legrange</groupId>
+      <artifactId>mikrotik</artifactId>
+      <version>3.0.7</version>
+      <scope>compile</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.openhab.core.bundles</groupId>
+      <artifactId>org.openhab.core</artifactId>
+      <version>3.1.0-SNAPSHOT</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
This dependency needs to be removed to make the binding build.

Reply:
That helped, thank you @fwolter !